### DISCUSSION TITLE
[Feature Request] Bookmark issues feature + bookmarked-issues list

### DISCUSSION THREAD

**Discussion by ashuvssut** [community/community#16331] [Projects and Issues]
GitHub Issues sometimes provide better solutions to our errors/problems than StackOverflow. 

It would be wonderful If we could **bookmark those issue** and then later access them whenever we want from our "Bookmarked issues List"  or "Bookmarked issue-comments List"

**Comment by shiplu** [comment]
This is a good idea. Unfortunately, no one feels about it.
Come on give some thumbs up!

**Reply by omitobi** [reply]
Always hoping to have this functionality also.

**Comment by rickswe** [comment]
Or allow the filter in your Inbox to search for key words like '#81|#765' . Then you can can create a filter called
Bookmarked Issues.

**Comment by shivamka1** [comment]
Its 2024 and this feature is still missing!! Why?

**Comment by GoaMind** [comment]
Earlier opened issue related to same topic: https://github.com/orgs/community/discussions/12369

**Comment by qwadrox** [comment]
For this specific reason, I created a Chrome extension: https://chromewebstore.google.com/detail/github-issue-glance/kallgfejdofmpelngmpklcddomojplod?authuser=0&hl=en

<img width="977" alt="image" src="https://github.com/user-attachments/assets/554208cb-2f2f-49b2-be2c-03f0bac2e233" />

It allows you to bookmark and manage bookmarked issues and that's it.

**Reply by ghost** [reply]
@qwadrox How do you use this extension once it is enabled? I don't see anything changing on the Github user interface.

**Reply by qwadrox** [reply]
@chelgeson1click There were some breaking changes on Github's issue pages. In the next release I fixed them. [V1.1.0](https://github.com/qwadrox/github-issue-glance/tree/refs/tags/V1.1.0)

**Comment by imyhxy** [comment]
I think **Bookmark/Save/Archive** is a **MUST-HAVE** feature, but nothing there in the Github. 😞

**Comment by jo-chemla** [comment]
Bookmarking issues, but also pulls, would indeed be great, as well as finding bookmarked issues/pulls to keep track of which ones are important to give a reply, test etc. 

Also paired to this, saving drafts not only locally, but syncing across github web, android app etc, and having a centralized way to access these drafts to continue working on them before posting comments (to both issues or pulls) would be great. Could not find the dedicated community thread for drafts comments on issues/pulls feature.
